client/settings/views/services/entry.js

```diff
@@ -23,23 +25,24 @@ const ShippingServiceEntry = ( props ) => {
 	const hasError = errors[ service.id ];
 
 	return (
-		<div className={ classNames( 'wcc-shipping-service-entry', { 'wcc-error': hasError } ) }>
+		<div className={ classNames( 'wcc-shipping-service-entry', { 'wcc-error': hasError } ) } title={ disabled ? __( 'You can manage this service after enabling the corresponding package in the Packaging Manager' ) : '' }>
 			<label className="wcc-shipping-service-entry-title">
 				<FormCheckbox
-					checked={ enabled }
+					checked={ enabled && ! disabled }
```

Comment:
Please come up with more descriptive variable names. I realize that they are technically correct, but this line hurts my brain.